tests/src/mint.rs

```diff
@@ -247,47 +247,32 @@ fn mint_should_return_dictionary_key_to_callers_owned_tokens() {
 
     let account = builder.get_expected_account(*DEFAULT_ACCOUNT_ADDR);
 
-    let receipt: String = query_stored_value(
+    let nft_receipt: String = support::query_stored_value(
         &mut builder,
-        nft_contract_hash,
+        nft_contract_key,
         vec![RECEIPT_NAME.to_string()],
     );
 
-    let (_, owned_tokens_key) = account
+    let account_receipt = *account
         .named_keys()
-        .get_key_value(&receipt)
-        .expect("should have owned_tokens_key");
-
-    match builder.query(None, *owned_tokens_key, &[]).unwrap() {
-        casper_types::StoredValue::CLValue(val) => {
-            let expected = val
-                .into_t::<Vec<u64>>()
-                .expect("should be Vec<u64> as Identifier defaults to indices");
-            assert_eq!(vec![0u64], expected);
-        }
-        _ => panic!("wrong stored value type"),
-    }
-
-    let mint_session_call = ExecuteRequestBuilder::standard(
-        *DEFAULT_ACCOUNT_ADDR,
-        MINT_SESSION_WASM,
-        runtime_args! {
-            ARG_NFT_CONTRACT_HASH => nft_contract_hash,
-            ARG_TOKEN_OWNER => Key::Account(*DEFAULT_ACCOUNT_ADDR),
-            ARG_TOKEN_META_DATA => TEST_PRETTY_721_META_DATA.to_string(),
-        },
-    )
-    .build();
-
-    builder.exec(mint_session_call).expect_success().commit();
+        .get(&format!("{}-m-{}-p-{}", nft_receipt, PAGE_SIZE, 0))
```

Comment:
maybe make a test utility to produce the formatted name

Reply:
Extracted the to a common function